sql/hive/pom.xml

```diff
@@ -238,36 +238,13 @@
              <configuration>
                <sources>
                  <source>v${hive.version.short}/src/main/scala</source>
-                 <source>${project.build.directory/generated-sources/antlr</source>
+                 <source>${project.build.directory}/generated-sources/antlr</source>
```

Comment:
Yeah you are right about that. So this could actually be removed. Lemme know, I can also submit a PR for this.